### PR TITLE
track stats about addon/file status values

### PR DESCRIPTION
**Pull request by kumar303**
This begins to address some stats needed like:
* how many addons have been submitted?
* how many addons are waiting for review?

### CODE REVIEW

**Comment by mstriemer** [comment]
Code looks good. Do we want something in more of a log format as well? Maybe with some data associated with the addon? I guess we could add that later but it seems like we could generate graphs from that log if we had it.

**Comment by kumar303** [comment]
statsd feeds into graphite so we can generate all kinds of time series graphs. Here's an example: [the last 15 minutes of AMO requests](http://dashboard.mktadm.ops.services.phx1.mozilla.com/graphite?site=addons&graph=all-responses). These graphs are old so I need to configure some more and fix up the graphing site (or maybe make a new one). I think this patch should allow us to graph how many add-ons are submitted in a given time period and how many are approved, rejected, deleted, etc.

File: apps/addons/models.py [hunk 1]
```diff
@@ -2320,3 +2321,24 @@ def update_incompatible_versions(sender, instance, **kw):
 models.signals.post_delete.connect(update_incompatible_versions,
                                    sender=CompatOverrideRange,
                                    dispatch_uid='cor_update_incompatible')
+
+
+def track_new_status(sender, instance, *args, **kw):
+    if kw.get('raw'):
+        # The addon is being loaded from a fixure.
```
Comment (magopian) [inline apps/addons/models.py]: I know I'm late to the party, but there's a typo here: fixTure

File: apps/amo/models.py [hunk 2]
```diff
@@ -240,6 +240,11 @@ def watch_status(old_attr={}, new_attr={},
                     new_instance.save(_signal=False)
             TheModel.on_change(watch_status)
 
+        ``old_atr`` will be a dict of the old instance attributes.
+        ``new_attr`` will be a dict of the new instance attributes, including
+        any that had not been changed by the operation that triggered the
+        callback (such as an update only of one field).
+
```
Comment (magopian) [inline apps/amo/models.py]: :+1:

**Pull request merged**